packages/app-mobile/utils/shim-init-react/shimInitShared.ts

```diff
@@ -76,7 +76,7 @@ const shimInitShared = () => {
 	};
 
 	shim.mobilePlatform = () => {
-		return Platform.OS;
+		return Platform.OS as MobilePlatform;
```

Comment:
An alternative that doesn't use the `as MobilePlatform` cast might be:

```suggestion
		if (Platform.OS === 'ios') return MobilePlatform.Ios;
		if (Platform.OS === 'web') return MobilePlatform.Web;
		if (Platform.OS === 'android') return MobilePlatform.Android;
		return MobilePlatform.None;
```